templates/terraform/examples/autoscaler_basic.tf.erb

```diff
@@ -46,7 +46,14 @@ resource "google_compute_instance_group_manager" "foobar" {
   name = "<%= ctx[:vars]['igm_name'] %>"
   zone = "us-central1-f"
 
+<% if ctx[:version].nil? || ctx[:version] == 'ga' -%>
```

Comment:
The examples/generated tests use `ctx` because that's just how `compile_file` (the compile command that compiles those in `custom_code.rb`) works- you pass it a hash of variables that you want the file to have access to. For whatever reason, it's different from `compile_files`, which is what the other stuff that gets compiled uses.